src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -136,7 +136,7 @@
             </trans-unit>
             <trans-unit id="37">
                 <source>This is not a valid IP address.</source>
-                <target>Det här är inte en giltig IP-adress.</target>
+                <target>Det här är inte en giltig IP-address.</target>
```

Comment:
Looks like you changed if because of fabbot, but it does not speak Swedish, so I suppose this should be reverted.

Reply:
Sure.. Yes, they were suggested by fabbot. Let me revert.